frontend/pom.xml

```diff
@@ -45,6 +45,11 @@
       <artifactId>auto-value-annotations</artifactId>
       <version>${auto-value.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value</artifactId>
```

Comment:
Do you need this?  Feel free to say yes, but I thought you only needed the above auto-value-annotations and the annotation processor.

Reply:
yeah, when I tried to build, it gave me a AutoValue_TripModel not found error which went away after I added this dependency

Reply:
Ah, I see, seems like there is a difference in behavior with an older version of maven compiler version this might be defaulting too (also the cause of a compiler warning). Essentially the version of maven you're defaulting to doesn't auto-pull in annotation processor dependencies. You can submit with this and I'll change it later or you can remove this and add the following:
`<version>3.8.1</version>`
after the line 
`<artifactId>maven-compiler-plugin</artifactId>`